### ISSUE TITLE
History & Search feature is not working on 1.6.0

### ISSUE THREAD

**Issue by Sangdol** [dbcli/mycli#258]
First of all, thank you for the great application.

After I upgraded to v1.6.0, `<C-r>` shortcut is not working anymore. It works well when I use v1.5.2. I read [changelog](https://github.com/dbcli/mycli/blob/master/changelog.md) but I couldn't find something related to this? Am I missing something?

**Comment by amjith** [comment]
You're right this was broken in the latest release. I'll fix it soon.

**Comment by michaelstack** [comment]
I have also noticed this has stopped working in vi mode with `ESC /<searchterm>`

@amjith I am guessing that you already fixed vi mode in https://github.com/dbcli/mycli/pull/259 ?

**Comment by amjith** [comment]
Yes it is fixed. You can try it by installing directly from master as follows: 

```
pip install -U https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by michaelstack** [comment]
@amjith I installed from master. I can confirm that it is working for me again. 

Thanks.

**Comment by tsroten** [comment]
I'm closing this issue since it was released in version 1.7.0.

**Comment by Sangdol** [comment]
Great thanks!
